hs_communities/views/communities.py

```diff
@@ -0,0 +1,138 @@
+from __future__ import absolute_import
+
+import json
+
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.models import Group, User
+from django.shortcuts import render
+from django.utils.decorators import method_decorator
+from django.utils.html import mark_safe, escapejs
+from django.views.generic import TemplateView
+
+from hs_access_control.management.utilities import community_from_name_or_id
+from hs_access_control.models.community import Community
+from hs_communities.models import Topic
+
+
+class CollaborateView(TemplateView):
+    template_name = 'pages/collaborate.html'
+
+
+class CommunityView(TemplateView):
+    template_name = 'hs_communities/community.html'
+
+    def dispatch(self, *args, **kwargs):
+        return super(CommunityView, self).dispatch(*args, **kwargs)
+
+    def get_context_data(self, **kwargs):
+        # user_id = User.objects.get(pk=self.request.user.id)
+        grpfilter = self.request.GET.get('grp')
+
+        community_resources = community_from_name_or_id("CZO National Community").public_resources
+        groups = []
+        for c in community_resources:
+            if not any(str(c.group_id) == g.get('id') for g in groups):  # if the group id is not already present in the list
+                if c.group_name != "CZO National":  # The National Group is used to establish the entire Community
+                    res_count = len([r for r in community_resources if r.group_name == c.group_name])
```

Comment:
The first community is for CZO only and code will be generalized, when the second Community comes online

Reply:
I'm not sure what this means... Is there no infrastructure yet for a user to create a community?

Reply:
The requirements for the next Community haven't been gathered yet. Doesn't make sense to generalize until you get the 2nd and 3rd data points...

Reply:
I disagree.  It doesn't make sense to have communities built in a way that requires a programmer to add.

Reply:
I'm with @sblack-usu on this one.

Reply:
Let's be more specific about the implementation change being requested. I'm reading your minds and guessing you want this to accept a community id integer and display that community, please confirm or refine.

Reply:
I was just responding to your comments, specifically "the requirements for the next Community haven't been gathered yet"

As for code changes, I don't think it's practical to create the infrastructure for a user to create/manage communities in this PR in our limited time window.  But I do think it's reasonable to remove the assumption that a community is CZO, by updating this view to take a path parameter that identifies the community.